Global/JetBrains.gitignore

```diff
@@ -50,17 +22,12 @@ out/
 # JIRA plugin
 atlassian-ide-plugin.xml
 
-# Cursive Clojure plugin
-.idea/replstate.xml
-
 # Crashlytics plugin (for Android Studio and IntelliJ)
 com_crashlytics_export_strings.xml
 crashlytics.properties
 crashlytics-build.properties
 fabric.properties
 
-# Editor-based Rest Client
-.idea/httpRequests
+# Pycharm virtual environment
+venv
```

Comment:
A broad set of `venv`-related rules is covered in `Python.gitignore` which should avoid the need for this rule:

https://github.com/github/gitignore/blob/7079791ab6058c544dd8dde60294b69e02a6dcf8/Python.gitignore#L98-L105